docs/reference/method/MongoDBClient__construct.txt

```diff
@@ -47,6 +57,48 @@ demonstrates how to connect to a replica set with a custom read preference:
        ]
    );
 
+Connecting with SSL and Authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example demonstrates how to connect to a MongoDB replica set with
```

Comment:
A link to [Connection String URI Format](https://docs.mongodb.com/manual/reference/connection-string/) was already in the parameter documentation, but I've also added it do the general "See Also" list at the bottom of the page.